protocol/messenger.go

```diff
@@ -2289,6 +2290,15 @@ func (m *Messenger) SendChatMessages(ctx context.Context, messages []*common.Mes
 
 // sendChatMessage takes a minimal message and sends it based on the corresponding chat
 func (m *Messenger) sendChatMessage(ctx context.Context, message *common.Message) (*MessengerResponse, error) {
+	m.logger.Debug("inside sendChatMessage",
```

Comment:
id, displayname,clock will all be blank at this stage, you want to log this at the bottom of the function

Reply:
thanks will ammend. 
Do we want any other field that may be necessary?
I'm currently verifying this branch with 2 builds on my local environment

Reply:
that's enough, I believe